architecture/adr-001-multi-denomination-rewards.md

```diff
@@ -0,0 +1,88 @@
+# ADR 001: Multi Denomination Rewards
+
+## Changelog
+
+- 2023-05-11: Initial Draft (@joe-bowman)
+
+## Status
+
+PROPOSED
+
+## Abstract
+
+With the advent of replicated security, delegators now accrue rewards in multiple
+denominations. These collected rewards should be distributed proportionally to the
+appropriate qAsset holders for the host zone.
+
+## Context
+
+The current implementation of rewards distribution is such that
+rewards collected in the host zone's staking denom are delegated, minus a
+parameterized fee that is sent, along with the remaining rewards (any non-staking-
+denom fees that have been collected) to the FeeCollectorAccount on Quicksilver to
```

Comment:
do we move the non-staking denoms to a different account and then distribute?

Reply:
Yes; the partial implementation [here](https://github.com/ingenuity-build/quicksilver/pull/427) uses the local account that corresponds to the withdrawal address. This saves us having to store another field, and is a value we know will not change over time.